src/Route.tsx

```diff
@@ -4,6 +4,7 @@
 
 import * as PropTypes from "prop-types";
 import * as React from "react";
+import * as shallowCompare from "react-addons-shallow-compare";
```

Comment:
In the documentation for https://www.npmjs.com/package/react-addons-shallow-compare they say that it's not maintained anymore and that we should use `PureComponent`s instead. Is there a specific reason we're specifically using it?

Reply:
Right. I don't know why I used the packages instead of just using PureComponent. 
There is one bug though that appears when using PureComponents (they are not re-rendered when context changes). I would fix this however with this change in the cargo apps:
https://github.com/jlabeit/impargo-apps/pull/3144/commits/a172a55b5c20235796b8da53259cde3845add644

Reply:
Later on we can change to the new context API which should work with PureComponents